src/services/platform.ts

```diff
@@ -0,0 +1,3 @@
+import { isPlatform } from '@ionic/react'
+
+export const isIos = () => isPlatform('ios')
```

Comment:
not used?

Reply:
yep, only added the tools for the moment and tested locally. Need to find the best spot where to use this